web/src/pages/starter-kits/[starterKitsUrl].tsx

```diff
@@ -5,23 +5,59 @@ import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
 import { Card } from "@/components/starter-kits/Card";
 import { StepInfo } from "@/components/starter-kits/StepInfo";
 import { getMergedConfig } from "@/contexts/configContext";
+import { modifyContent } from "@/lib/domain-logic/modifyContent";
 import { ROUTES } from "@/lib/domain-logic/routes";
 import { buildUserRoadmap } from "@/lib/roadmap/buildUserRoadmap";
 import { Roadmap, Task } from "@/lib/types/types";
 import analytics from "@/lib/utils/analytics";
-import { ProfileData, createEmptyProfileData } from "@businessnjgovnavigator/shared/index";
-import type { InferGetStaticPropsType } from "next";
+import { Industries, Industry } from "@businessnjgovnavigator/shared";
+import {
+  LookupIndustryById,
+  ProfileData,
+  createEmptyProfileData,
+} from "@businessnjgovnavigator/shared/index";
+import type { GetStaticPathsResult } from "next";
 import { useRouter } from "next/router";
 import { ReactElement } from "react";
 
-const General = (props: InferGetStaticPropsType<typeof getStaticProps>): ReactElement => {
+interface Props {
+  noAuth: boolean;
+  roadmap: Roadmap;
+  industry: Industry;
+}
+
+const StarterKitsPage = (props: Props): ReactElement => {
   const router = useRouter();
   const Config = getMergedConfig();
 
   const getTaskNamesForStep = (tasks: Task[], step: number): string[] => {
     return tasks.filter((task) => task.stepNumber === step).map((task) => task.name);
   };
 
+  const insertIndustryContent = (content: string): string => {
+    return modifyContent({
+      content,
+      condition: () => props.industry.id !== "generic",
+      modificationMap: {
+        industry: props.industry.name,
+      },
+    });
+  };
+
+  const insertRoadmapSteps = (content: string): string => {
+    return modifyContent({
+      content,
+      condition: () => true,
+      modificationMap: {
+        numberRoadMapSteps: String(props.roadmap.steps.length),
+      },
+    });
+  };
```

Comment:
[sand] These functions can probably exist outside the React function component, no?

Reply:
Sure

Reply:
Actually I think it might be better if they stayed in, because they access props with their content and we'd have to pass in a bunch of extra params for every call.

If you feel real strong I can do that though

Reply:
I don't have as much context for where it would be reusable but if it would be I don't think it would be too bad, as they each only access a single value from props if I'm reading it correctly. Seems like it might be a good opportunity to create a `web/src/helpers/content.tsx` or some such file

Only other tiny nit here would be I think `getIndustryContent` and `getRoadmapSteps` might be more descriptive of what's happening here as opposed to `insert`